### PR TITLE
Define Endpoint Formats in README

### PR DESCRIPTION
**Pull request by emichaud998**
Changes in this PR:
This PR defines the CERNER and NPPES endpoint formats the same way that we already do for Epic's format in the endpoint manager README file. I also have added a section to the README that describes how to add a new endpoint list.

Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: [Lantern-223 JIRA Ticket](https://oncprojectracking.healthit.gov/support/browse/LANTERN-223)
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [x] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

### CODE REVIEW

File: endpointmanager/README.md [hunk 1]
```diff
@@ -202,5 +204,28 @@ The Endpoint Manager expects the format of an endpoint source list to be in the
...
+      "name": <name of the organization>,
+      "baseUrl": <location of the FHIR endpoint>,
+      "type": <endpoint type>
+    },
+    ...
+  ]
+}
+```
+
+NPPES:
```
Comment (mayerm94) [inline endpointmanager/README.md:222]: Name Which NPPES file this represents

Comment (mayerm94) [inline endpointmanager/README.md:222]: Should Also add file format (JSON or CSV) next to the source

File: endpointmanager/README.md [hunk 2]
```diff
@@ -202,5 +204,28 @@ The Endpoint Manager expects the format of an endpoint source list to be in the
...
+
+```
+"NPI","Endpoint Type","Endpoint Type Description","Endpoint","Affiliation","Endpoint Description","Affiliation Legal Business Name","Use Code","Use Description","Other Use Description","Content Type","Content Description","Other Content Description","Affiliation Address Line One","Affiliation Address Line Two","Affiliation Address City","Affiliation Address State","Affiliation Address Country","Affiliation Address Postal Code"
+...
+```
+
+### Adding a New Endpoint List
+
+To add a new endpoint list, add an entry to the EndpointResourcesList.json file located in the resources/prod_resources directory with the endpoint name, the name the endpoint source file will be saved as, and the endpoint URL. If the format does not match any of those listed above in the expected endpoint formats, add a new parser.
```
Comment (mayerm94) [inline endpointmanager/README.md:231]: Add more info about adding a new parser. eg: `See lantern-back-end/endpointmanager/pkg/fetcher/cernerlist.go or  lantern-back-end/endpointmanager/pkg/fetcher/epiclist.go for examples of the interface which endpoint list parsers need to adhere to.`
